content/docs/guides/neon-auth.md

```diff
@@ -64,6 +64,7 @@ The following columns are included in the `neon_auth.users_sync` table:
 - `email`: The user's primary email (nullable)
 - `created_at`: When the user signed up (nullable)
 - `deleted_at`: When the user was deleted, if applicable (nullable)
+- `updated_at`: When user info was last updated, if applicable (nullable)
```

Comment:
nit picking a bit but I will ask anyway :D 
why `user info` and not `the user` like in other columns?